### PR TITLE
Update the README file

### LINKED ISSUE #58: Update ReadMe
## Description

Update and rewrite the README with clear instructions how to setup nourish.

## Requirements

- [ ] Update README
- [ ] Add Docker to README

### PR DESCRIPTION
**Pull request by Anissa3005**
# Description

Previously, our README file was cluttered with unnecessary information. I have cleaned up the README and update it with new information

# Closes issue(s)

Resolves #58 

# How to test

NA

# Changes include

1. Add setup for Docker
2. Removed unnecessary information

# Checklist
- [x] I have updated the README

### CODE REVIEW

**Review by oddpharmacy — APPROVED** [review]
Lgtm :)

**Review by FefesG5 — APPROVED** [review]
Looks good to me. I had to check the preview to see what is kept in the readme. The github comparison is difficult to see.

File: README.md [hunk 1]
```diff
@@ -2,140 +2,95 @@
...
+npm run prepare
 ```
 
-2. Module Installation
+#### 3. Setup Database 🐳
 
-```sh
-npm install
-```
+To keep in simple we use Docker to run an image of PostgreSQL database.
```
Comment (shonancanuck) [inline README.md:49]: Super minor grammar/spelling:
To keep *in* simple
suggested: it, things

File: README.md [hunk 2]
```diff
@@ -2,140 +2,95 @@
...
 
-3. Creating a database with Postgres
+#### 3.a Add database URL to your .env file
 
-- Mac:
+This should match with the Docker environment.
 
 ```sh
-psql postgres
+DATABASE_URL=postgresql://dbuser:passworddb@localhost:5432/namedb
```
Comment (shonancanuck) [inline README.md:56]: This is the url for a local instance. We can probably provide the url for the docker instance?

**Review by shonancanuck — CHANGES_REQUESTED** [review]
Looks good! Two small requests.

**Review by shonancanuck — APPROVED** [review]

**Pull request merged**